packages/api/cli/src/util/terminate.ts

```diff
@@ -6,10 +6,17 @@ function redConsoleError(msg: string) {
 
 process.on('unhandledRejection', (reason: string, promise: Promise<unknown>) => {
   redConsoleError('\nAn unhandled rejection has occurred inside Forge:');
-  redConsoleError(reason.toString());
+  redConsoleError(reason.toString().trim());
```

Comment:
These changes were done in this PR but could maybe be extracted, I was annoyed by the awful promise rejection logging so I fixed it 🤷